src/Refactoring-UI/StRefactoringPreviewPresenter.class.st

```diff
@@ -114,30 +113,25 @@ RBRefactoringPreviewPresenter >> initializeLayout [
 		                add: scopeDropList;
 		                yourself.
 	actionsLayout := SpBoxLayout newLeftToRight
-		                 addLast: buttonOk
-		                 expand: false
-		                 fill: true
-		                 padding: 5;
-		                 addLast: buttonCancel
-		                 expand: false
-		                 fill: true
-		                 padding: 5;
+		                 spacing: 5;
+		                 addLast: buttonOk expand: false fill: true;
+		                 addLast: buttonCancel expand: false fill: true;
 		                 yourself.
 
-	self layout: (SpBoxLayout newTopToBottom
-			 add: (SpBoxLayout newTopToBottom
-					  add: scopesLayout
-					  withConstraints:
-						  [ :constraints | constraints height: 30 ] yourself;
-					  add: table;
-					  add: diffPresenter;
-					  yourself);
-			 add: actionsLayout expand: false;
-			 yourself)
+	^ self layout: (SpBoxLayout newTopToBottom
```

Comment:
There is only one thing left to do, and that is to return the layout without assignment in.
So, you only need to remove the `self layout: (...)` something like this:

```
^ (SpBoxLayout newTopToBottom
    add: (SpBoxLayout newTopToBottom
    ...
```